router/middlewares/middlewareLogger.go

```diff
@@ -5,11 +5,16 @@ import (
 	"net/http"
 
 	"github.com/ditrit/badaas/configuration"
+	"github.com/gorilla/mux"
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.